_posts/2020_04_29_encrypted_keras_with_syft/encrypted_keras_with_syft.Rmd

```diff
@@ -0,0 +1,413 @@
+---
+title: "Towards privacy: Encrypted deep learning with Syft and Keras"
+description: > 
+  Deep learning need not be irreconcilable with privacy protection. Federated learning enables on-device, distributed model training; encryption keeps model and gradient updates private; differential privacy prevents the training data from leaking. As of today, private and secure deep learning is an emerging technology. In this post, we introduce Syft, an open-source framework that integrates with PyTorch as well as TensorFlow. In an example use case, we obtain private predictions from a Keras model.  
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanaprivacysyftkeras
+bibliography: bibliography.bib
+date: 04-29-2020
+categories:
+  - R
+  - Privacy & Security
+  - TensorFlow/Keras
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+The word *privacy*, in the context of deep learning (or machine learning, or "AI"), and especially when combined with things
+like *security*, sounds like it could be part of a catch phrase: *privacy, safety, security* -- like *liberté, fraternité,
+égalité*. In fact, there should probably be a mantra like that. But that's another topic, and like with the other catch phrase
+just cited, not everyone interprets these terms in the same way.
+
+So let's think about privacy, narrowed down to its role in training or using deep learning models, in a more technical way.
+Because privacy -- or rather, its violations -- may appear in various ways. Different violations will demand different
+countermeasures. Of course, in the end, we'd like to see them all integrated -- but here's a journey where people are just
+starting to embark. The most important thing we can do, then, is to learn about the concepts, sight the landscape of
+implementations under development, and -- perhaps -- decide to join the effort.
+
+This post tries to do a tiny little bit of all of those.
+
+## Aspects of privacy in deep learning
+
+Say you work at a hospital, and would be interested in training a deep learning model to help diagnose some disease from brain
+scans. Where you work, you don't have many patients with this disease; moreover, they tend to mostly be affected by the same
+subtypes: Your training set, were you to create one, would not very well reflect the overall distribution. It'd thus make
+sense to cooperate with other hospitals; but that isn't so easy, as the data collected is protected by privacy regulations.
+So, the first requirement is: The data has to stay where it is; e.g., it may not be sent to a central server.
```

Comment:
I would emphasize the requirement somehow (perhaps italicize it), so it's not lost in the scenario description.